### DISCUSSION TITLE
Is it possible to do simple oidc group authorization without rbac?

### DISCUSSION THREAD

**Discussion by michaelschmit** [kiali/kiali#6600] [General]
I was curious if it is possible to do simple oidc group authorization without rbac?  Since the group membership can be returned from oidc is there a way to require the user to be in a certain group to access Kiali?

**Comment by jmazzitelli** [comment]
You can disable RBAC with this setting: https://kiali.io/docs/configuration/kialis.kiali.io/#.spec.auth.openid.disable_rbac

> Since the group membership can be returned from oidc is there a way to require the user to be in a certain group to access Kiali?

Not that I know of, but I am not all that familiar with the openid integration.
